README.md

```diff
@@ -1 +1,38 @@
-# angular-multiple-controllers-lab
\ No newline at end of file
+# Multiple Controllers
+
+## Objectives
+
+- Write a new Controller
+- Nest the second Controller inside the other in the View
+- Access inherited properties from the parent Controller
+- Update a parent Controller from the child Controller
+
+## Instructions
+
+Create your directory structure as follows:
+
+- js/
+  - app/
+    - controllers/
+      - ContactController.js
```

Comment:
the file that's included is called "MainController' - if they aren't going to use it in this lab, I would leave it out.